central/sensor/service/connection/upgradecontroller/trigger.go

```diff
@@ -129,11 +129,11 @@ func (u *upgradeController) doTriggerUpgrade() (common.MessageInjector, *central
 	case storage.ClusterUpgradeStatus_SENSOR_VERSION_HIGHER:
 		// We still allow upgrade triggers in this case.
 	case storage.ClusterUpgradeStatus_MANUAL_UPGRADE_REQUIRED:
-		return nil, nil, errors.Errorf("manual upgrade required for cluster %s; cannot trigger upgrade", u.clusterID)
+		return nil, nil, errors.Errorf("RHACS does not manage version for secured cluster %s; cannot trigger upgrade/downgrade", u.clusterID)
 	case storage.ClusterUpgradeStatus_UP_TO_DATE:
 		return nil, nil, errors.Errorf("sensor for cluster %s is already up-to-date; cannot trigger upgrade", u.clusterID)
 	default:
-		return nil, nil, errors.Errorf("unknown upgradability status of sensor for cluster %s; cannot trigger upgrades", u.clusterID)
+		return nil, nil, errors.Errorf("unknown upgradability status of sensor for cluster %s; cannot trigger upgrade/downgrade", u.clusterID)
```

Comment:
```suggestion
		return nil, nil, errors.Errorf("The sensor's upgradability status for cluster %s is unclear; cannot trigger upgrade or downgrade", u.clusterID)
```

Reply:
The message is actually for the secured cluster. Sensor was used as a name, because we used to call secured clusters like this. Maybe I would change it to secured cluster?

Reply:
Addressed in [f81eac1](https://github.com/stackrox/stackrox/pull/12833/commits/f81eac1c3bb3705d1a08dd84fbc5e019187b768f) - I changed sensor to secured cluster and rephrased a bit.